account.go

```diff
@@ -817,6 +855,8 @@ type AccountCapabilities struct {
 	TaxReportingUS1099MISC AccountCapabilityStatus `json:"tax_reporting_us_1099_misc"`
 	// The status of the transfers capability of the account, or whether your platform can transfer funds to the account.
 	Transfers AccountCapabilityStatus `json:"transfers"`
+	// The status of the US bank account ACH payments capability of the account, or whether the account can directly process US bank account charges.
+	USBankAccountAchPayments AccountCapabilitiesUSBankAccountAchPayments `json:"us_bank_account_ach_payments"`
```

Comment:
Double checking - should this be `USBankAccountACHPayments`?

Reply:
Ahhh yes it should be. Will fix now

Reply:
Ah actually, it's `Ach` everywhere else right now, so I'd rather keep it as is until we can fix it everywhere. Is that cool?